internal/pkg/githubapi/pr_metrics.go

```diff
@@ -0,0 +1,108 @@
+package githubapi
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v62/github"
+	lru "github.com/hashicorp/golang-lru/v2"
+	log "github.com/sirupsen/logrus"
+	prom "github.com/wayfair-incubator/telefonistka/internal/pkg/prometheus"
+)
+
+const (
+	timeToDefineStale = 20 * time.Minute
+	metricRefreshTime = 60 * time.Second
+)
+
+func MainGhMetricsLoop(mainGhClientCache *lru.Cache[string, GhClientPair]) {
+	for t := range time.Tick(metricRefreshTime) {
+		log.Debugf("Updating pr metrics at %v", t)
+		getPrMetrics(mainGhClientCache)
+	}
+}
+
+func getRepoPrMetrics(ctx context.Context, ghClient GhClientPair, repo *github.Repository) (prWithStaleChecks int, openPRs int, openPromotionPrs int, err error) {
```

Comment:
Since we pass the first three output of `getRepoPrMetrics` function directly to `PublishPrMetrics`.
Maybe we can move those to a struct type, so we can have more readable function signature.